### DISCUSSION TITLE
Models

### DISCUSSION THREAD

**Discussion by OlliSant** [a-cosmology-group/acg#30] [Metaphysics and Philosophy]
The list of models http://cosmology.info/essays/models_marmet.html 

The list is good to be here for a start. There are more than 80 models in the scientific community or near it. They are not all refuted. They are refuted only if we first suppose the BB model as right. And Louis has also made an article where he values all the models with some college. 

In my mind the universe is eternal and infinite, and the philosophy is better than physics to study the universe. Cosmology is always both philosophy and astronomy, in this order. We must understand the universe, the eternity and the infinity better now with all new astronomy of the galaxies.

No expansion, no beginning. The local part- universe has a beginning. The observable universe is stupid thing for a definition of the universe, the universe is much bigger.

**Comment by genotics** [comment]
Hi Olli, When I was an archaeology student in Germany many decades ago, my fellow students used to mock the American tendency to speak in terms of models, whereas what they were referring to would traditionally have been termed theories. By now, what was once newspeak has engulfed most disciplines. Yet we do not refer to the general model of relativity, or the model of quantum mechanics. These retain the theory label, even in the popular TV series. If the term model has a place in science, it could charitably be understood as a mathematical structure, i.e., something at a lesser level of abstraction and generality than a theory. And this is they way I generally conceive of a model, i.e., a phenomenological description rather than an explanatory conceptualization.
WIthin this framework, a terminology is needed to establish clearly the object of any discussion. If we can agree that the object at the highest level of generalization is most likely infinite and eternal, which we customarily call the universe or cosmos, then the finite portion to which we are limited in our discussions might be referred to by the term which Toivo Jaakola used in his work, and probably derived from old Soviet terminology, i.e., metagalaxy. This has the advantage of being more compact than terms like visible universe, and expresses the notion that the range of our theorizations is tightly restricted. 
So let us forge ahead toward a viable theory of the metagalaxy. Or we just stick with cosmology.
Cheers, Roy

**Reply by OlliSant** [reply]
Yes! The terms must be defined, they must be clear. Those entities we are handling in cosmology are the **whole universe** and all different kinds of **parts** in the whole universe. The universe must mean the universe, the whole universe, not the **observable universe**, which is a smaller and different entity. What we mean exactly by **metagalaxy**, still has this ambiguity. If it means all the galaxies and all the space, it has already some ontological presuppositions. These are the presuppositions that Toivo Jaakola and me are ready to make, I think. And all the space and galaxies in the observable universe or in the (whole) universe? 

Theory and model can be used to mean the same thing or they can be different things. In discussion of theories of the universe it does not matter. Some other things are more important. Is the theory physical, mathematical, philosophical or theological? BB is mathematical- physical. It misses the philosophy, and is theologically ignoristic. My theory, proposition, is philosophical, but we can make physical mathematical theories of it, 4D ball- like structure and Q for example, where Q is the longest possible distance in the reality, the diameter in the space of the universe.

Some people here say that there is not such thing as the universe, the whole universe. I think it exists, but differently as any other things.

**Reply by OlliSant** [reply]
And the classical physics or the GR can be changed to the 4D and Q model. That goes farther in mathematics than my competence, but I can imagine how it can be made. There you must help me if you agree with the view of the universe.

**Comment by OlliSant** [comment]
I show you how to make a theory of the universe in philosophy. It is much better as the mathematical theories of the cosmologists.

**Three possibilities**

There are three logical possibilities for a right theory of the universe. Steady State theories, Multiverse theories and Cyclic theories. It is easy to make a good theory! Just take wrong things away from each theory, and leave the right points there, and unite them after that. This can be made in many ways, this is one way.

1. So, we get from the Steady State that there is an ultimate steady state in the universe, it is static, but which kind of a state it is in the reality of the universe?

2. Multiverse says there are different beginnings, and so different galaxy populations. No parallel universes, just this only one universe. Parallel universes are only part- universes like the galaxy populations too are only part- universes.

3. Cycles are local too. No cycle for the universe. The universe has its steady state.

There we have a theory. The proof is that there are galaxies far away, and there are galaxies near.  **Universe is the web of the galaxies.** This web is eternal and infinite, when we define eternal and infinite in the right way, corresponding to the reality. The universe is static, the local galaxy populations and galaxies and stars have their dynamics. The mathematical- physical theories are missing for these kinds of philosophical theories. That has happened because of the dictatorship of the BB- theory, the expansion. And the dictatorship of physics and mathematics over philosophy. And the dictatorship of empiricism and naturalism over other philosophies.

Where is God? Asks theologians. We can ignore God in the physics of the universe, because universe is not created, it has no beginning and it has no birth. If there is a creation, it is for earth for example. God is a supreme being in the universe, not the creator of it from nothing. If he exists.

It means a lot for the astronomy of the universe if he exists. But not for the physics of the universe, for the physics of the cosmology. The eternal and infinite universe is a materialistic theory. The existence of God means, that there are higher beings too as humans, eternal beings, and different habitable planets as our earth. But the basic physics of the universe might be the same in both cases.

And it means there are spirit worlds too in the universe and worlds of God, paradises. Unknown things now for our science.

**Reply by RedshiftDrift** [reply]
*Steady State* is not the same theory as *Static*.

**Steady state** accepts expansion of space, while there is no expansion of space in a **static** model.

**Reply by OlliSant** [reply]
I don't mean the Steady State theory. I mean there ultimately is a static steady state in the universe. Then it is not expanding. The Steady State theory is only one of the Steady state- theories. The expansion must be taken away from the Steady state- theories, and then they fit better with the reality of the universe.
